### ISSUE TITLE
YouTube not blocked on Chrome

### ISSUE THREAD

**Issue by jlaw9** [aniketpanjwani/chomper#46]
**Note: for support questions, please use [Gitter](https://gitter.im/chomperapp/Lobby)**. This repository's issues are reserved for feature requests and bug reports.

* **Do you want to request a *feature* or report a *bug*?**
bug

* **What is the current behavior?**
chomper is blocking sites on my blocklist, except for YouTube on Chrome. If I try to open youtube in incognito mode or Firefox, it is blocked, but for some reason, it isn't blocked in my regular chrome account. I tried disabling all of my extensions, but that didn't have an effect.
I looked at the screen session running under sudo, and nothing is showing up for youtube.

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** 
`chomper noyoutube 5`
Open youtube.com on Chrome in incognito mode and it is blocked. Open it in regular mode and it isn't blocked.

* **Please tell us about your environment:**
  
  - Chomper Release or Commit: latest
  - Browser:  Chrome Version 69.0.3497.100 (Official Build) (64-bit)
  - Operating System: Ubuntu 18.04.1

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

Thanks in advance for help!

**Comment by v-marinkov** [comment]
I can confirm this behaviour. However, I have found a workaround. To stop videos from loading on any browser block:
      - googlevideo.com

Keep the blocking on youtube.com for all non-chromium browsers too.

If you still find the Youtube feed distracting, I advise you to install the [Distraction Free YT extension](https://chrome.google.com/webstore/detail/df-tube-distraction-free/mjdepdfccjgcndkmemponafgioodelna?hl=en) and customize it to hide almost everything but the search bar. Although it is easy to turn it off, I find that the clean look of Youtube makes me appreciate how distracting it otherwise is and I even get repulsed by its look now, so I end up never turning off the extension.

For good measure I also blocked everything related to youtube according to PiHole but it doesn't make any noticeable difference:
      - youtube-ui.l.google.com
      - ytimg-edge-static.l.google.com
      - ytstatic.l.google.com
      - ytimg.l.google.com
      - i1.ytimg.com
      - ytimg.com
      - youtube-ui.l.google.com


Good luck!
